xarray/core/computation.py

```diff
@@ -1587,15 +1587,6 @@ def cross(
     array([-3,  6, -3])
     Dimensions without coordinates: dim_0
 
-    Vector cross-product with 2 dimensions, returns in the perpendicular
-    direction:
```

Comment:
might also need to change the description in the docstring, a bit further up? It still says 1, 2, or 3 dimensional vectors are supported.